.ci/load/Jenkinsfile

```diff
@@ -75,6 +79,15 @@ pipeline {
                                 setEnvVar('JAVA_HOME', sh(script: ".ci/load/scripts/fetch_sdk.sh ${params.jvm_version}", returnStdout: true).trim())
                                 setEnvVar('JAVACMD', "${env.JAVA_HOME}/bin/java")
                                 setEnvVar('PATH', "${env.JAVA_HOME}/bin:$PATH")
+
+                                def jdk_major = sh(script: ".ci/load/scripts/java_major.sh ${params.jvm_version}", returnStdout: true).trim())
+                                def app_build_jdk = "${env.JAVA_HOME}"
+                                if(jdk_major == '7'){
+                                    // build application with a more recent JDK that don't have TLS issues
+                                    app_build_jdk = sh(script: ".ci/load/scripts/fetch_sdk.sh zulu-8.0.272-linux", returnStdout: true).trim()
```

Comment:
I am worried about this line because I think it has the potential to overwrite the JDK that we download here on L79.

Reply:
do you think of any better way to handle this ? We could maybe have a `java7` folder alongside the `java11` that is used to build the agent.

Reply:
I wonder if we could just generate have the script generate a randomly generated directory name so we can be (fairly) sure we're avoiding collisions? So, we'd modify the shell script to return `java-7-mYRand0mStr`? Either way, so long as it's unique, I think we'll be fine.